src/Bridge/Doctrine/EventListener/PublishMercureUpdatesListener.php

```diff
@@ -150,6 +150,7 @@ private function storeEntityToPublish($entity, string $property): void
             $this->deletedEntities[(object) [
                 'id' => $this->iriConverter->getIriFromItem($entity),
                 'iri' => $this->iriConverter->getIriFromItem($entity, UrlGeneratorInterface::ABS_URL),
+                'type' => $this->resourceMetadataFactory->create($this->getObjectClass($entity))->getShortName(),
```

Comment:
`$this->getObjectClass($entity)` is already known as `$resourceClass` line 123
And the metadata were also already created line 128 and can be re-used I guess

Reply:
good catch :)